packages/react-select/src/Select.tsx

```diff
@@ -605,7 +652,6 @@ export default class Select<
   commonProps: any; // TODO
   initialTouchX = 0;
   initialTouchY = 0;
-  instancePrefix = '';
```

Comment:
I moved instancePrefix  to state to be able to use it inside of getDerivedStateFromProps